lib/ansible_spec/load_ansible.rb

```diff
@@ -287,4 +288,60 @@ def self.get_properties()
     end
     return properties
   end
+
+  def self.get_variables(host, group_idx)
+    vars = {}
+    p = self.get_properties
+
+    # all group
+    vars_file = 'group_vars/all.yml'
+    if File.exist?(vars_file)
+      yaml = YAML.load_file(vars_file)
+      if yaml.kind_of?(Hash)
+        vars.merge!(yaml)
+      end
+    end
+
+    # each group vars
+    if p[group_idx].has_key?('group')
```

Comment:
I missed group handling when refactor the code.
I fixed and add lacked role default variable handling.

thanks,